controllers/articles-controllers.js

```diff
@@ -0,0 +1,15 @@
+const { fetchArticleById } = require('../models/articles-models');
+
+exports.getArticleById = (req, res, next) => {
+  const { article_id } = req.params;
+
+  fetchArticleById(article_id)
+    .then((article) => {
+      res.status(200).send({ article });
+    })
+    .catch((err) => {
+      res.status(err.status).send({ msg: err.msg });
+      console.log(err.msg, '<-----HERE')
```

Comment:
So the way we want to deal with any errors is with error handling middleware, and this would usually be implemented in `app.js`, and then in this controller you would just need to invoke `next()` with the error.

So go back and have a look at error handling here: [https://notes.northcoders.com/courses/js-back-end/error-handling](url)